README.md

```diff
@@ -65,6 +65,29 @@ The app automatically launches with the Chromium developer tools opened by defau
 
 Code formatting has been set up to make merging PRs easier. It uses the same prettier/eslint mechanism as Calypso. See [JavaScript Coding Guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/javascript.md) for details on setting up your editor.
 
+### Testing
+
+#### Running tests
```

Comment:
I wonder if we could also mention that with the command `npm test -- --watch` contributors can run specific tests, which might be useful when developing. WDYT?

Reply:
Actually, we have a specific command for this: `npm run test:watch`.